docs/releases.rst

```diff
@@ -1,6 +1,333 @@
+- to prevent crendential bruteforce (an attacker trying to guess a password), we limit the maximal amount of attempts an attacker has for a given account (from any IP address)
+- to prevent password spraying (an attacker trying the same common password on all accounts he can enumerate), we limit the maximal number of non-existing accounts an attacker can attempt to authenticate against from a given network subnet.
+
+We have also implemented state-of-the-art features such as `Device Cookies <https://owasp.org/www-community/Slow_Down_Online_Guessing_Attacks_with_Device_Cookies>`_ and IP-whitelisting post-authentication to ensure we don't lock genuine users out.
+
+Rate-limiters have a bad name because they are often misunderstood. If you used Mailu's rate-limiter in the past and had a bad experience please consider giving it another try after upgrading.
+
+Remember the login URL
+^^^^^^^^^^^^^^^^^^^^^^
+
+Mailu will now remember which URL was requested and redirect you to it post-authentication.
+
+This functionality can be used by visiting a "deep" URL E.g.
+
+- https://test.mailu.io/admin
+- https://test.mailu.io/webmail
+
+This results in a login page with a single login button. To access the normal login page, visit the root url.
+
+- https://test.mailu.io
+
+Users who only use the /admin endpoint can now bookmark https://test.mailu.io/admin. When logging in, it is possible to use the `Enter` key again.
+
+Introduction of SnappyMail
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Rainloop webmail client has been replaced with SnappyMail.
+The Rainloop project has multiple long outstanding security bugs. For this reason the Mailu project looked for alternatives.
+SnappyMail is a fork of Rainloop focussed on performance and security. It offers a similar experience as Rainloop.
+
+Do not mark spam as read
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the user settings it is now possible to configure if a received spam email must be marked as read.
+It is  possible to see if you received spam now.
+
+OLETools
+^^^^^^^^
+
+`OLETools <https://github.com/decalage2/oletools>`_ is introduced to block bad macros in Microsoft Office documents. OLETools is able to scan Microsoft Office documents and determine if a macro is malicous.
+
+By default attachments with know bad/executable file extensions (such as ``.exe``) are blocked. See the FAQ for more information on updating the list of blocked file extensions.
+
+New override system for Rspamd
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The override system for Rspamd has been overhauled. While the config files were first completely overridden, they are now merged.
+Now overrides are placed in the location (in the Rspamd/Antispam container) /overrides.
+
+If you use your own map files, change the location to /override/myMapFile.map in the corresponding conf file.
+It works as following.
+
+* If the override file overrides a Mailu defined config file,
+  it will be included in the Mailu config file with lowest priority.
```

Comment:
I assume lowest priority means the override will take precedence to the Mailu defaults? This could be clarified what it means